cumulus/pallets/xcmp-queue/src/lib.rs

```diff
@@ -341,11 +341,16 @@ pub mod pallet {
 	pub(super) type OutboundXcmpStatus<T: Config> =
 		StorageValue<_, Vec<OutboundChannelDetails>, ValueQuery>;
 
-	// The new way of doing it:
-	/// The messages outbound in a given XCMP channel.
+	/// The messages outbound in a given XCMP channel. This is the old queue with `u16` indices.
```

Comment:
We should create an issue for revisiting this and removing this storage type once it no longer has keys, right?

Reply:
Not sure if we already have an established process for this, but since this pallet is going to be replaced soon by the message queue pallet, I think it might not be necessary. Otherwise, we can check back in 3-4 months after this PR is deployed.